### ISSUE TITLE
`rake tmp:clear` did not clear my cache

### ISSUE THREAD

**Issue by nwallace** [ai/autoprefixer-rails#101]
I'm using Rails 4.2.6 and for some reason, running `rake tmp:clear` does not clear my cache, and so the CSS was not prefixed. I had to run the more specific `rake tmp:cache:clear` before my CSS would include the prefixes. Might be worth it to be more specific in the README, or to suggest using this command as a fallback.

**Comment by ai** [comment]
Sure. Do you want to send PR?

**Comment by Myrronth** [comment]
I'm using Rails 4.2.7 and updated autoprefixer-rails from 6.3.7 to 6.4.0. After restarting my local dev-server, the css doesn't include any prefixes anymore. This seems to only happen in development mode. My staging server works fine. `rake tmp:clear`and `rake tmp:cache:clear` hasn't helped. Any suggestions?

**Comment by ai** [comment]
@Myrronth show your input/output CSS

**Comment by Myrronth** [comment]
My CSS is written in sass and split up in dozens of files. What's the best way to give you the source? Is one section of SASS-Code (that includes properties that should be prefixed) enough?

source: https://gist.github.com/Myrronth/abb05d826db7a202da1644f8bc5bb9dc
output: https://gist.github.com/Myrronth/2ccfdc9b98aea3b1d8f1854c100dfcbe

mask-image should be prefixed with -webkit-

**Comment by ai** [comment]
@Myrronth BTW, Autoprefixer supports `@supports`, you don’t need to write prefixes there.

**Comment by ai** [comment]
Could you test `mask` property outside `@supports`?

**Comment by Myrronth** [comment]
Sure! I've updated the gists. The problem still exists outside of `@supports`

Source: https://gist.github.com/Myrronth/abb05d826db7a202da1644f8bc5bb9dc
Output: https://gist.github.com/Myrronth/2ccfdc9b98aea3b1d8f1854c100dfcbe

**Comment by ai** [comment]
@Myrronth did you have it in development mode too (when you visit website in browser)?

**Comment by Myrronth** [comment]
Yes, it happens in all environments (development and staging). My previous assumption that it works in staging is wrong, because the gem was updated but the css wasn't recompiled.

**Comment by ai** [comment]
@Myrronth OK, you need to debug this lines (clean cache before run):

1. That Autoprefixer runned https://github.com/ai/autoprefixer-rails/blob/master/lib/autoprefixer-rails/processor.rb#L20
2. That Autoprefixer is installed https://github.com/ai/autoprefixer-rails/blob/master/lib/autoprefixer-rails/sprockets.rb#L36
3. That Rails find Autoprefixer https://github.com/ai/autoprefixer-rails/blob/master/lib/autoprefixer-rails/railtie.rb#L11-L21

**Comment by ai** [comment]
@Myrronth your issue is here #102

**Comment by ai** [comment]
@nwallace do you want to send PR? :)

**Comment by Myrronth** [comment]
Sorry, I was on the move. I upgraded to 6.4.0.1 and that solved my problem. Thanks!

**Comment by nwallace** [comment]
@ai Thanks for your patience!  I ran into some more issues, and I wanted to diagnose them first before I got back to you.

After my initial problem where I thought `rake tmp:cache:clear` fixed my issue, my prefixing stopped working again (just in development -- my testing environment was working fine). I don't recall exactly what I did to break it, but it continued being broken these past few weeks.

Today I tried to address the issue again, and it didn't seem to get fixed until I shut down my rails server, then ran `rake tmp:clear`, then started my rails server again.  Unfortunately I'm having trouble reproducing it, so it's hard to say for sure that shutting down the server first was necessary to fix my problem, but that's the way it seemed.

My issue seems to be resolved now after whatever combination of things I did to fix it, so this can probably be closed. Thanks!
